rules/windows/execution_via_xp_cmdshell_mssql_stored_procedure.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution via MSSQL xp_cmdshell stored procedure. Because malicious users sometimes attempt to elevate their
+privileges by using xp_cmdshell, xp_cmdshell is disabled by default. Thus it's important to review the context of it's
+use.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Execution via MSSQL xp_cmdshell stored procedure"
+risk_score = 73
+rule_id = "4ed493fc-d637-4a36-80ff-ac84937e5461"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

```suggestion
event.category:process and event.type("start" or "process_started") and
```

for process events, I would be explicit be using `"start"` because there is also an event.type of `info` which may result in unrelated or duplicative alerts. `process_started` is for backwards compat for pre 7.8 beats